app/helpers/application_helper.rb

```diff
@@ -187,6 +187,14 @@ def courses
     ]
   end
 
+  def custom_button
```

Comment:
Custom button isn't a very descriptive name, can we change this to something else please?